### PR TITLE
Completed seed data script

### PR DESCRIPTION
**Pull request by Weswwn**
Completed server API endpoint

Manually tested API endpoint and received correct data back from database

Had to reconfigure webpack due to unexpected issue

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -9,9 +10,17 @@ class App extends React.Component {
     };
   }
 
+  componentDidMount() {
+    axios.get('/api/restaurants/?/reviews', {
+      params: {
+        // r_id: 1,
```
Comment (howardyang25) [inline client/src/App.jsx]: Is there a reason this is commented out?

Comment (Weswwn) [reply client/src/App.jsx]: I haven't coded in the part where we grab the restaurant from the actual localhost:3000 url yet. I did testing from postman to confirm that the server can talk to the database and grab the correct data. I'll complete that part after this pull is merged.

Thanks!

File: db/dataGenerator.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+};
+
+const genReviewData = () => {
+  const numberOfReviews = 5;
+  const masterReviewArray = [];
+  for (let z = 1; z < 101; z++) {
+    const eachRestReviewData = [];
+    for (let x = 1; x < numberOfReviews; x++) {
+      const obj = {
+        overallRating: Math.floor(Math.random() * 5) + 1,
```
Comment (howardyang25) [inline db/dataGenerator.js]: You can also just go Math.floor(Math.random() * 6) to achieve the same thing

Comment (Weswwn) [reply db/dataGenerator.js]: Ohh, will do that next time. Thanks for the tip!

File: server/Server.js [hunk 3]
```diff
@@ -1,15 +1,27 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 const db = require('../db/dbConnection.js');
 
 const app = express();
 const port = 3000;
 
 // app.get('/', (req, res) => res.send('Hello World!'))
 
+app.use(bodyParser.json());
```
Comment (howardyang25) [inline server/Server.js]: app.use(express.json()) will achieve the same thing without having to require body parser, not sure about urlencoded, would have to consult docs

File: server/Server.js [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static('public'));
 
 app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```
Comment (howardyang25) [inline server/Server.js]: Minor suggestion: take out "example"?

File: server/Server.js [hunk 5]
```diff
@@ -1,15 +1,27 @@
...
 
-const queryString = 'INSERT INTO users(username, location, numberOfReviews) VALUES(?, ?, ?)';
+// console.log(dataGenerator.genUserData());
 
-db.connection.query(queryString, ['bob', 'San Francisco', 100]);
+app.get('/api/restaurants/:id/reviews', (req, res) => {
+  const queryString = 'SELECT * from reviews where r_id = ?';
+  db.connection.query(queryString, [req.params.id], (error, results, fields) => {
```
Comment (howardyang25) [inline server/Server.js]: if you're not using fields, you could not include it as a param

Comment (Weswwn) [reply server/Server.js]: Will do!

**Pull request merged**